### ISSUE TITLE
no-unsupported-features allows object spread on Node v6 even though object spread is not yet supported

### ISSUE THREAD

**Issue by coltonw** [mysticatea/eslint-plugin-node#85]

**Comment by mysticatea** [comment]
Thank you for this issue.

Object rest/spread properties are not a part of JavaScript yet.
That rule reports JavaScript syntax only.

**Comment by mysticatea** [comment]
If you disallow object rest/spread properties, please use default parser of ESLint.

**Comment by coltonw** [comment]
Ah, I see!  I was extending the airbnb rules and they turn on the experimental object spread. For future Googlers, I just had to add the following to my .eslintrc to have eslint reject the object spread as an error again:

```yaml
ecmaFeatures:
  experimentalObjectRestSpread: false
```
Closing the issue and thank you for your quick response!
